main.py

```diff
@@ -170,9 +196,15 @@ def download_videos(
         "simulate": True,
         "quiet": True,
         "cookiesfrombrowser": (COOKIES_FROM_BROWSER,),
+        "cookiefile": COOKIE_FILE,
         "format": "worstaudio"
     }
 
+    if COOKIE_FILE != None:
+        opts.pop("cookiesfrombrowser")
+    else:
+        opts.pop("cookiefile")
+
```

Comment:
Not sure what this does?

Reply:
It removes the key from opts dict

Reply:
Ok but why ?

Reply:
Ah to make sure they are exclusive...

In that case I suggest https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group so the code and `--help` are more readable

Reply:
But it doesn't help to remove the if block right?